great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -62,6 +62,8 @@ def get_user_friendly_error_message(
             errors = error_json.get("errors")
         if errors:
             support_message.append(json.dumps(errors))
+        else:
+            support_message.append(json.dumps(error_json))
```

Comment:
The flow here is somewhat confusing. If I'm reading this correctly, there are 3 ways we get errors?
* error_json is a list, so assign errors to error_json
* error_json is a dict with an errors list, so we pull them out
* error_json is the error itself, so we use that?
Why not make the first if/else clear about those flows (i.e. replace the else with an elif isinstance(dict), then add another else to say `errors = error_json`

Reply:
that was my first approach, but they've been typed in an artificial way (using `TypedDict`), and it would be more effort to rewrite. I think its pretty high priority to get these API errors to users, so just went with the most direct approach.

Reply:
Yes that TypedDict isn't accurate anymore.
Updating and standardizing the format of our DomainErrors is one of my current priorities.

Until that's done I don't think it's worth spending too much time on this, but we do need to make sure users can see some details about the errors.